var/spack/repos/builtin/packages/py-pytorch-lightning/package.py

```diff
@@ -72,6 +72,7 @@ class PyPytorchLightning(PythonPackage):
     depends_on("py-lightning-utilities@0.3", when="@1.8:", type=("build", "run"))
 
     # Historical dependencies
+    depends_on("py-lightning-lite@1.8.0", when="@1.8.0", type=("build", "run"))
```

Comment:
These bits were extracted out into a separate library in 1.8.0, but vendored starting in 1.8.1.